pkg/plugins/manager/loader/loader.go

```diff
@@ -309,6 +309,14 @@ func (l *Loader) readPluginJSON(pluginJSONPath string) (plugins.JSONData, error)
 		}
 	}
 
+	if plugin.Extensions != nil {
+		for _, link := range plugin.Extensions.Links {
+			if !filepath.IsAbs(link.Path) {
+				link.Path = fmt.Sprintf("/%s", link.Path)
+			}
+		}
+	}
+
```

Comment:
Question: where are we going to cater for making sure that the links are prefixed with `/a/<plugin-id>`?

Reply:
Aaa I just saw, it's done [on the frontend](https://github.com/grafana/grafana/pull/61663/files#diff-2f2dcc3ad31894679878a1b60a9dfa7cd3ccff4430a0a8c3f6425b2d4e31144aR31). Only throwing it up as an idea: would it make sense to do it in the backend instead so we have information coming down the pipe that we can use "straight away"?

Reply:
Our reasoning here was that the `/a/` route is a frontend route which the backend doesn't have knowledge about. So it feels like it is something for the frontend to construct. If that makes sense?

Reply:
Sure 👍